app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -152,15 +157,18 @@ module.exports = React.createClass
           <div key={selects[i].id}>
             {if selects[i].title isnt @props.task.instruction
               <div>{selects[i].title}</div>}
-            <Select
+            {if selects[i].allowCreate
+              SelectComponent = Select.Creatable
+            else
+              SelectComponent = Select}
+            <SelectComponent
               options={options}
               onChange={@onChangeSelect.bind(@, i)}
-              value={@props.annotation.value[i]?.value}
+              value={@state.value[i]?.value}
```

Comment:
`react-select` used to expand simple string and number values out to Option objects `{label, value}` but 1.0 removed that. I suspect this bug is due to values still being passed around as simple strings in some places.